qt@4.rb

```diff
@@ -66,6 +66,12 @@ class QtAT4 < Formula
     sha256 "5e81df9a1c35a5aec21241a82707ad6ac198b2e44928389722b64da341260c5d"
   end
 
+  # Patch to fix build on macOS Big Sur
+  patch do
+    url "https://raw.githubusercontent.com/qkdreyer/homebrew-qt4/98a906e1ad47106c65021150938de61138799ea8/patches/qt4-bigsur.patch"
```

Comment:
Could you please update this to say `/cartr/` instead of `/qkdreyer/` so it points at the original repository and not your fork?